src/runtime_src/core/edge/skd/sk_daemon.cpp

```diff
@@ -338,6 +338,10 @@ void configSoftKernel(xclSKCmd *cmd)
      * kernel image.
      */
     pid = fork();
+    if (pid > 0) {
```

Comment:
I think the rule is no "{" "}" for one line statement.

Reply:
done